Bio/pairwise2.py

```diff
@@ -1099,7 +1099,8 @@ def print_matrix(matrix):
                        for j in range(len(matrix[i]))))
 
 
-def format_alignment(align1, align2, score, begin, end):
+def format_alignment(align1, align2, score, begin, end,
+                     show_full_sequences=False):
```

Comment:
Would ``full_seq`` be enough as the variable name? That this is about displaying or showing seems clear to me from the context.

Reply:
OK. I changed that.